src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -559,9 +559,14 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       "sparse" -> (if (i % 2 == 0) AttributeNull else AttributeNumber(i.toDouble)),
       "vocab1" -> AttributeString(vocab1Strings(i % vocab1Strings.size)),
       "vocab2" -> AttributeString(vocab2Strings(i % vocab2Strings.size)),
-      "mixed" -> (i % 2 match {
+      "mixed" -> (i % 3 match {
         case 0 => AttributeString(s"$i")
         case 1 => AttributeNumber(i.toDouble)
+        case 2 => AttributeValueList( 1 to i map( AttributeNumber(_) ) reverse )
```

Comment:
I could shuffle for the numeric type, but shuffling the list in the alphabetic case would change its position relative to other column elements.